aspnetcore/security/anti-request-forgery.md

```diff
@@ -266,17 +267,58 @@ The following example uses JavaScript to make an AJAX request to obtain the toke
 > [!NOTE]
 > When the antiforgery token is provided in both the request header and in the form payload, only the token in the header is validated.
 
+<a name="afwma"></a>
+
 ### Antiforgery with Minimal APIs
 
-`Minimal APIs` do not support the usage of the included filters (`ValidateAntiForgeryToken`, `AutoValidateAntiforgeryToken`, `IgnoreAntiforgeryToken`), however, <xref:Microsoft.AspNetCore.Antiforgery.IAntiforgery> provides the required APIs to validate a request.
+Call [AddAntiforgery](/dotnet/api/microsoft.extensions.dependencyinjection.antiforgeryservicecollectionextensions.addantiforgery) <!-- <xref:Microsoft.Extensions.DependencyInjection.AntiforgeryServiceCollectionExtensions.AddAntiforgery(IServiceCollection)> --> to register antiforgery services in DI. `WebApplicationBuilder` automatically adds the middleware when the antiforgery services have been registered in the DI container. Antiforgery tokens are used to mitigate [cross-site request forgery attacks](xref:security/anti-request-forgery).
```

Comment:
@captainsafia  I fixed your suggestion from the other PR here. This is a copy/paste of the content in the other PR.